src/components/Undo/Undo.scss

```diff
@@ -0,0 +1,69 @@
+.undo {
+  -moz-box-align: center;
+  -moz-box-pack: center;
+  display: -moz-box;
+  left: 6px;
+  position: absolute;
+  top: 6px;
+  z-index: 1;
+  padding: 4px 3px;
+  border: 1px solid;
+  border-color: rgba(8,22,37,.12) rgba(8,22,37,.14) rgba(8,22,37,.16);
+  background-color: rgba(255,255,255,.4);
+  color: #525e69;
+  font-size: 11px;
+
+  #undo-label {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .undo-button {
+    -moz-appearance: none;
```

Comment:
It seems we now support the unprefixed version of `appearance`:
http://caniuse.com/#search=appearance

Be nice to drop the prefix for this.